src/segments/README.md

```diff
@@ -48,6 +48,21 @@ truncation/compacting for that specific segment.
   shows the git branch and current status, set ´SEGMENTS_GIT_BRANCH_ONLY=true´
   to speed up execution on large repos. The default value is ´false´.
 
+  Enother option to speed up execution is to use gitstatus as a backend.
+  Install gitstatus and then add the following to your .bashrc_:
```

Comment:
Do we still require sbp to start an interactive shell with this approach?

Reply:
Yes, https://github.com/romkatv/gitstatus/blob/master/gitstatus.plugin.sh#L3